lang/de-DE.json

```diff
@@ -0,0 +1,53 @@
+{
+    "Abbe Number": "Abbe-Zahl",
+    "Albedo": "Albedo",
+    "Anisotropy": "Anisotropie",
+    "Anisotropy Rotation": "Anisotropierotation",
+    "Attenuation Color": "Dämpfungsfarbe",
+    "Attenuation Distance": "Dämpfungsdistanz",
+    "Base": "Basis",
+    "Basic": "Standard",
+    "Car Paint": "Autolack",
+    "Clearcoat": "Klarlack",
+    "Clearcoat Normal": "Klarlacknormale",
+    "Clearcoat Roughness": "Klarlackrauheit",
+    "Coating": "Beschichtung",
+    "Cut-Out Opacity": "Deckkraft",
+    "Displacement": "Reliefstruktur",
+    "Emission": "Emission",
+    "Emission Color": "Emissionsfarbe",
+    "Emission Mode": "Emissionsmodus",
+    "Emission Value": "Emissionsstärke",
+    "Emissive": "Leuchtend",
+    "Energy Normalization": "Energienormalisierung",
+    "Enterprise PBR Shading Model": "Enterprise PBR Shading Model",
+    "Flake Color": "Partikelfarbe",
+    "Flake Coverage": "Partikeldichte",
+    "Flake Roughness": "Partikelrauheit",
+    "Flake Size": "Partikelgröße",
+    "Flakes": "Partikel",
+    "Flip Flop": "Interferenz",

```

Comment:
We should not use "Interferenz" here. We may introduce actual interference at some point, so this will cause a name clash/confusion. Why not use "Flipflop/Flipflopfarbe"? Those are commonly accepted terms in Germany.

Reply:
Thanks for the valuable input @bsdorra. I adjusted accordingly!